apps/vr-tests-react-components/src/stories/Positioning.stories.tsx

```diff
@@ -1084,6 +1084,47 @@ const ScrollJumpContext = () => {
   );
 };
 
+const MultiScrollParent = () => {
+  const { targetRef, containerRef } = usePositioning({});
+  const scrollContainerRef = React.useRef<HTMLDivElement>(null);
+
+  const scroll = () => {
+    if (scrollContainerRef.current) {
+      scrollContainerRef.current.scrollBy({ top: 100 });
+    }
+  };
+
+  return (
+    <>
+      <span>The popover should stay attached to the trigger</span>
+      <button id="scroll" onClick={scroll}>
+        scroll
+      </button>
+      <div ref={scrollContainerRef} style={{ border: '2px dashed green', height: 300, width: 400, overflow: 'auto' }}>
+        <div style={{ height: 5000, width: 300, position: 'relative' }}>
+          <div
+            style={{
+              overflow: 'auto',
+              border: '2px dashed red',
+              position: 'absolute',
+              top: 150,
+              left: 100,
+              padding: 20,
+            }}
+          >
+            <button id="target" ref={targetRef}>
+              Trigger
+            </button>
+          </div>
+        </div>
+      </div>
+      <div ref={containerRef} style={{ border: '2px solid blue', padding: 20, backgroundColor: 'white' }}>
+        Popover
+      </div>
+    </>
+  );
+};
```

Comment:
```suggestion
const MultiScrollParent = () => {
  const { targetRef, containerRef } = usePositioning({});
  const scrollContainerRef = React.useRef<HTMLDivElement>(null);

  const scroll = () => {
    if (scrollContainerRef.current) {
      scrollContainerRef.current.scrollBy({ top: 150 });
    }
  };

  return (
    <>
      <span>The popover should stay attached to the trigger</span>
      <button id="scroll" onClick={scroll}>
        scroll
      </button>
      <div
        ref={scrollContainerRef}
        style={{
          border: '2px dashed green',
          height: 300,
          width: 400,
          overflow: 'auto',
          display: 'grid',
          gridTemplateColumns: '350px auto',
          gridTemplateRows: '800px',
        }}
      >
        <div style={{ position: 'relative' }}>
          <div
            style={{
              overflow: 'auto',
              border: '2px dashed red',
              position: 'absolute',
              top: 150,
              left: 100,
              padding: 20,
            }}
          >
            <button id="target" ref={targetRef}>
              Trigger
            </button>
          </div>
        </div>
        <div
          style={{
            backgroundImage: 'linear-gradient(to bottom, #ffffff, #b9b9b9, #777777, #3b3b3b, #000000)',
          }}
        />
      </div>
      <div ref={containerRef} style={{ border: '2px solid blue', padding: 20, backgroundColor: 'white' }}>
        Popover
      </div>
    </>
  );
};
```

Let's make the story clearer and an indicator for the scroll (please check that it still catches the regression)